app/views/exclusion/exclusionNinoOrNoNinoForm.scala.html

```diff
@@ -24,7 +24,7 @@
 @import uk.gov.hmrc.play.views.html.helpers
 @import utils.URIInformation._
 
-@parentTemplate(pageTitle = Messages("ExclusionNinoDecision.title") + " - " + Messages("Service.title"), Some(taxYearRange), Some(empRef.toString) ) {
+@parentTemplate(pageTitle = Messages("ExclusionNinoDecision.title") + " - " + Messages("Service.title"), Some(taxYearRange), Some(empRef.toString), showLanguage = true) {
```

Comment:
Do we still need the show language flag if it is shown on every page? Could this not be refactored out?

Reply:
There are still some pages that doesn't have the welsh toggle on, not sure about the business logic behind it. Otherwise happy to refactor it on the parentTemplate so it doesn't need to be passed in from child views.